build/Dockerfile.base

```diff
@@ -1,21 +1,15 @@
-FROM docker.io/nvidia/cuda:11.8.0-base-ubi8
+FROM registry.access.redhat.com/ubi8/ubi:8.4
 
-ARG ARCH=amd64
-
-# enable centos appstream
-RUN yum install -y http://mirror.centos.org/centos/8-stream/BaseOS/aarch64/os/Packages/centos-gpg-keys-8-6.el8.noarch.rpm && \
-    yum install -y http://mirror.centos.org/centos/8-stream/BaseOS/aarch64/os/Packages/centos-stream-repos-8-6.el8.noarch.rpm
-
-# don't update yet, this will conflict with protected pkg i.e. redhat-release
-#RUN yum update -y
-
-# install less frequently updated pkg first
-RUN yum install -y kmod xz python3 && yum clean all -y && \
+RUN yum update -y && \
+#    yum install linux-headers-`uname -r` && \
+    yum install -y kernel-devel linux-headers-`uname -r` curl kmod xz python3 && yum clean all -y && \
```

Comment:
@rootfs , https://github.com/sustainable-computing-io/kepler/actions/runs/4455303181/jobs/7825038254?pr=587#step:6:1371

it seems when we use ubi image, it will mount host vm's kenerl version. and 
```
linux-headers-`uname -r `
```
will points to a not existing linux-header rpm package as `linux-headers-5.15.0-1034-azure`